src/net/download.rs

```diff
@@ -149,10 +157,40 @@ impl BlockRequestKey {
             child_block_header: child_block_header,
             sortition_height: sortition_height,
             download_start: get_epoch_time_secs(),
+            kind,
+        }
+    }
+}
+
+impl Requestable for BlockRequestKey {
+    fn get_url(&self) -> &UrlString {
+        &self.data_url
+    }
+
+    fn get_request_type(&self, peer_host: PeerHost) -> HttpRequestType {
```

Comment:
Should probably be `make_request_type` instead of `get_request_type` (since you're allocating a new structure, instead of accessing a member field).

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2013/commits/0e477d074b6c1ec7b88c6798b3f7def3500a8e2d